SpatialGDK/Source/SpatialGDK/Private/Tests/RPCServiceTest.cpp

```diff
@@ -1,11 +1,18 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 
 #include "CoreMinimal.h"
+#include "Engine/Engine.h"
+#include "GameFramework/GameStateBase.h"
+#include "Interop/SpatialReceiver.h"
 #include "Interop/SpatialRPCService.h"
 #include "Interop/SpatialStaticComponentView.h"
+#include "Misc/AutomationTest.h"
 #include "Schema/RPCPayload.h"
 #include "SpatialConstants.h"
 #include "SpatialGDKSettings.h"
```

Comment:
Can you drive-by split these GDK / engine imports into separate groups? (worker on using clang-format and it sorts per group so files like this will get rekt)